static/assets/js/index.js

```diff
@@ -89,7 +89,7 @@
   function showFormSuccess(form) {
     // TODO
     $(form)[0].reset();
-    setFormMessage(form, 'All set, we received your information!');
+    setFormMessage(form, 'All set, we received your information!  Look for your first text/email from us soon.');
```

Comment:
Since we won't get social sharing going for awhile, let's modify so it reads: All set, we received your information. Look for your first text/email from us soon, and please help us spread the word by sharing Calls for Change on social!